test/testactivitylistmodel.cpp

```diff
@@ -0,0 +1,416 @@
+/*
+ * Copyright (C) by Claudio Cambra <claudio.cambra@nextcloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+#include "gui/tray/activitylistmodel.h"
+
+#include "account.h"
+#include "accountstate.h"
+#include "accountmanager.h"
+#include "syncenginetestutils.h"
+#include "syncresult.h"
+
+#include <QAbstractItemModelTester>
+#include <QDesktopServices>
+#include <QSignalSpy>
+#include <QTest>
+
+constexpr auto STARTING_ID = 90000;
```

Comment:
@claucambra With all my personal preference towards having a different case for **consts**, we have them with a **camel case** like every normal variable everywhere else in the code base.

Reply:
@mgallien What do you think?^^

Reply:
@allexzander yes please use normal camel case for this one to be consistent with other constants in tests

Reply:
Got it, thanks